pachyderm/values.yaml

```diff
@@ -109,8 +109,8 @@ imageCredentials: {}
 # move under ingress? How to make work with cert-manager, maybe add
 # tls option to pachctl create secret?
 tls:
-  crt:
-  key:
+  crt: ""
```

Comment:
How do you feel about `nil` for unspecified, versus empty string? Either can ultimately work, of course.

Reply:
I don't have strong opinions one way or the other (they were about half and half, so added the remainder for consistency)